src/backend/pluginPlan.test.js

```diff
@@ -116,4 +116,87 @@ describe("src/backend/pluginPlan", () => {
       expect(identity.contractGraph).toBeCalledTimes(0);
     });
   });
+
+  describe("pluginGraphPlan", () => {
+    it("should update discourse graph", async () => {
+      // Given
+      const loaders = mockPluginLoaders();
+      const cache = mockCacheProvider();
+      const ghToken = null;
+      const project = createProject({
+        id: "has-discourse",
+        discourseServer: {serverUrl: "http://foo.bar"},
+      });
+
+      // When
+      const createGraph = pluginGraphPlan(loaders, ghToken, cache);
+      const graph = await createGraph(project);
+
+      // Then
+      const {discourse, github, identity} = loaders;
+      expect(graph).toEqual(discourseGraph());
+      expect(discourse.updateMirror).toBeCalledTimes(0);
+      expect(discourse.createGraph).toBeCalledTimes(1);
+      expect(discourse.createGraph).toBeCalledWith(
+        project.discourseServer,
+        cache
+      );
+      expect(github.updateMirror).toBeCalledTimes(0);
```

Comment:
As discussed here (https://github.com/sourcecred/sourcecred/pull/1571#discussion_r368328923), I think that verifying that unrelated methods were not called is not robust to future changes, so we may as well not try to verify it.